### DISCUSSION TITLE
Can't log in. An error ocured while sending the request.

### DISCUSSION THREAD

**Discussion by Snairo** [ppy/osu#25298] [Q&A]
When i tried to log in the Message "An error ocured while sending the request." pops up. I don't know how to fix it. I restartet my pc disabled my Firewall un-and reinstaled osu!lazer but nothing worked.
[database.log](https://github.com/ppy/osu/files/13199380/database.log)
[performance.log](https://github.com/ppy/osu/files/13199381/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13199382/runtime.log)
[updater.log](https://github.com/ppy/osu/files/13199383/updater.log)
[input.log](https://github.com/ppy/osu/files/13199384/input.log)
[network.log](https://github.com/ppy/osu/files/13199385/network.log)

**Comment by peppy** [comment]
https://www.google.com/search?client=safari&rls=en&q=0x80090326&ie=UTF-8&oe=UTF-8

Your system SSL stack is likely compromised or corrupted.

**Reply by Snairo** [reply]
do yo know a way to fix that?

**Reply by peppy** [reply]
Please check the google link for potential solutions.
